packer/amazon/centos-puppet-agent.json

```diff
@@ -26,7 +26,7 @@
       "ssh_username": "centos",
       "ssh_pty": "true",
       "encrypt_boot": "{{user `ebs_volume_encrypted`}}",
-      "ami_name": "Tarmak CentOS 7 x86_64 with puppet-agent {{isotime \"2006-01-02_030405\"}}",
+      "ami_name": "{{user `tarmak_environment`}} - Tarmak CentOS 7 x86_64 with puppet-agent {{isotime \"2006-01-02_030405\"}}",
```

Comment:
Is that change necessary?

Reply:
No, change came from rebase, will revert